model-optimizer/mo/ops/pooling.py

```diff
@@ -52,10 +52,10 @@ def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, {
             'type': self.op,
             'op': self.op,
-            'version': 'opset1',
+            'version': 'opset8',
             'infer': self.infer,
             'in_ports_count': 1,
-            'out_ports_count': 1,
+            'out_ports_count': 1 if attrs.get('version') == 'opset1' else 2
         }, attrs)
```

Comment:
using atts you can define for which operation this constructor is called for `avg` or `maxpool`.